src/Css/Style.php

```diff
@@ -1025,9 +1029,8 @@ protected function single_length_in_pt(string $l, float $ref_size = 0, ?float $f
 
         $font_size = $font_size ?? $this->__get("font_size");
 
-        $calcPattern = "/^calc\((.*)?\)$/i";
-        if (preg_match($calcPattern, $l, $calcMatches)) {
-            return $this->evaluate_func_calc($this->parse_calc($calcMatches[1]), $ref_size, $font_size);
+        if (preg_match(self::CSS_MATH_FUNCTION_PATTERN, $l)) {
```

Comment:
I would generalize the pattern to any function, since the code below is only concerned with single values and `evaluate_func` already returns `null` for unsupported functions.